source/Launchpad-Commands/LaunchpadDryRunProcessingContext.class.st

```diff
@@ -0,0 +1,18 @@
+Class {
```

Comment:
You may create a new category for the processing contexts

Reply:
Also, why ProcessingContext?

LaunchpadApplicationContext
- DryRunLaunchpadApplicationContext / LaunchpadApplicationDryRunContext
- LaunchpadApplicationStartingContext

Reply:
This package doesn't have any tags (but the new Tonel export version in Pharo 12 duplicates that info from the package)